buildSrc/src/main/groovy/com/uber/okbuck/core/model/java/JavaLibTarget.groovy

```diff
@@ -15,20 +15,28 @@ class JavaLibTarget extends JavaTarget {
         super(project, name)
     }
 
+    protected Set<File> getMainSrcDirs() {
+        return project.sourceSets.main.java.srcDirs as Set
+    }
+
+    protected Set<File> getTestSrcDirs() {
+        return project.sourceSets.test.java.srcDirs as Set
+    }
+
     @Override
     Scope getMain() {
         return Scope.from(project,
-                compileConfigs,
-                project.sourceSets.main.java.srcDirs as Set,
+                depsConfig,
+                mainSrcDirs,
                 project.file("src/main/resources"),
                 project.compileJava.options.compilerArgs as List)
     }
 
     @Override
     Scope getTest() {
         return Scope.from(project,
-                expand(compileConfigs, TEST_PREFIX, true),
-                project.sourceSets.test.java.srcDirs as Set,
+                expand(depsConfig, UNIT_TEST_PREFIX),
```

Comment:
we can remove this expanding behavior since we can ask the java plugin for the singular configuration name similar to what we do for android

Reply:
removed for ones which have constants in JavaPlugin. still need to keep it for apt.